pilot/helpers/agents/Developer.py

```diff
@@ -35,26 +35,37 @@ def __init__(self, project):
         self.debugger = Debugger(self)
 
     def start_coding(self):
-        self.project.current_step = 'coding'
-        update_app_status(self.project.args['app_id'], self.project.current_step)
+        if self.project.finished:
+            development_plan = self.project.feature_development_plan
+        else:
+            self.project.current_step = 'coding'
+            development_plan = self.project.development_plan
+            update_app_status(self.project.args['app_id'], self.project.current_step)
 
-        if self.project.skip_steps is None:
-            self.project.skip_steps = False if ('skip_until_dev_step' in self.project.args and self.project.args['skip_until_dev_step'] == '0') else True
+            if self.project.skip_steps is None:
+                self.project.skip_steps = False if ('skip_until_dev_step' in self.project.args and self.project.args['skip_until_dev_step'] == '0') else True
 
         # DEVELOPMENT
         print(color_green_bold("🚀 Now for the actual development...\n"))
         logger.info("Starting to create the actual code...")
 
-        for i, dev_task in enumerate(self.project.development_plan):
+        for i, dev_task in enumerate(development_plan):
             self.implement_task(i, dev_task)
 
         # DEVELOPMENT END
         self.project.dot_pilot_gpt.chat_log_folder(None)
-        logger.info('The app is DONE!!! Yay...you can use it now.')
-        print(color_green_bold("The app is DONE!!! Yay...you can use it now.\n"))
+        if not self.project.finished:
```

Comment:
Does `project.finished` not mean what I expect it would mean?

```suggestion
        if self.project.finished:
```

Reply:
project.finished is set once we are completely done with development and go to project.finish() but i see why this could be missleading, I will update it

Reply:
So `project.finished` means that the first sprint / MVP is complete, but not _finished_?

I'd expect `project.finished` to be set to true if the user presses ENTER to declare that the project is "finished".
...but even then, the user should be able to come back in a day/week/month/year and say "hey, one more thing...".

Can we change it to `if self.project.sprint == 0` or `if self.project.current_step != 'coding'`?

Reply:
Yeah, you are right, project.finished means that MVP or in other words initial project is finished. Option to add more features/improvements is always there but it doesn't mean that project is not finished.

Probably down the line we will have to change that when we add new functionalities but I think we can leave it the way it is for now